dkg-gadget/src/lib.rs

```diff
@@ -176,15 +171,23 @@ where
 	let keygen_handle = tokio::spawn(keygen_gossip_handler.run());
 	let signing_handle = tokio::spawn(signing_gossip_handler.run());
 
+	// In memory backend, not used for now
+	// let db_backend = Arc::new(db::DKGInMemoryDb::new());
```

Comment:
Do you think we should extract this into a CLI flag? What is the benefit of using in memory.. is it just for testing? Seems like we should be able to test with offchain storage directly and simplify our approaches. No need to carry around extra baggage imo.